frontend/summary/summaryTable/editing/Root.js

```diff
@@ -10,27 +10,38 @@ import DjangoForm from "./DjangoForm";
 @observer
 class Root extends Component {
     render() {
-        const {tableStore, tableObject} = this.props.store,
+        const {tableStore, tableObject, isCreate, handleSubmit, cancelUrl} = this.props.store,
+            saveBtnText = isCreate ? "Create" : "Update",
             Form = getEditTableComponent(tableObject),
             Table = getViewTableComponent(tableObject);
 
         return (
-            <Tabs>
-                <TabList>
-                    <Tab>Overall</Tab>
-                    <Tab>Editing</Tab>
-                    <Tab>Preview</Tab>
-                </TabList>
-                <TabPanel>
-                    <DjangoForm />
-                </TabPanel>
-                <TabPanel>
-                    <Form store={tableStore} />
-                </TabPanel>
-                <TabPanel>
-                    <Table store={tableStore} forceReadOnly={true} />
-                </TabPanel>
-            </Tabs>
+            <>
+                <Tabs>
+                    <TabList>
+                        <Tab>Overall</Tab>
+                        <Tab>Editing</Tab>
+                        <Tab>Preview</Tab>
+                    </TabList>
+                    <TabPanel>
+                        <DjangoForm />
+                    </TabPanel>
+                    <TabPanel>
+                        <Form store={tableStore} />
+                    </TabPanel>
+                    <TabPanel>
+                        <Table store={tableStore} forceReadOnly={true} />
+                    </TabPanel>
+                </Tabs>
+                <div className="form-actions">
```

Comment:
can we make this div a reusable react component?  Maybe we can call it a `FormAction`?

It looks like there are currently 3 that already exist `BaseFormVisualReact, App, and DjangoForm`

The default for saveBtnText can be "Save"

Reply:
I found another case, `frontend/mgmt/TaskTable/containers/Root.js`, where I was able to replace the form actions. I also replaced all the other instances, except for `BaseFormVisualReact`; this belongs to the aggregate endpoints visual form, which I was not able to test due to existing errors.